src/cli/domain/init-template/install-template.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Spinner = require('cli-spinner').Spinner;
+const spawn = require('cross-spawn');
+const path = require('path');
+const _ = require('lodash');
+const process = require('process');
+const strings = require('../../../resources');
+
+module.exports = function installTemplate(options, scaffold) {
+  const {
+    componentName,
+    templateType,
+    compiler,
+    cli,
+    componentPath,
+    local,
+    template,
+    logger,
+    callback
```

Comment:
Not sure I follow you here. The signature is indeed `(option, callback)`. `scaffold` is not in the options cause its been passed as callback